ReleaseProcessAutomation/Steps/PipelineSteps/ContinueReleasePatchStep.cs

```diff
@@ -92,9 +92,28 @@ public void Execute (SemanticVersion nextVersion, bool noPush, bool onMaster)
 
     GitClient.Checkout(mergeTargetBranchName);
 
+    if (!onMaster)
+    {
+      CreateNewSupportBranch(nextVersion);
+    }
+
+    GitClient.Checkout(mergeTargetBranchName);
+    
     if (noPush)
       return;
 
     _pushPatchReleaseStep.Execute(mergeTargetBranchName, tagName, toMergeBranchName);
   }
+
+  private void CreateNewSupportBranch (SemanticVersion nextVersion)
+  {
+    Console.WriteLine("Do you wish to create a new support branch?");
```

Comment:
Console-Outout would also be a good abstraction to test that the correct messages are written and to prevent console spam during tests.